pkg/logql/sketch/quantile.go

```diff
@@ -38,8 +42,17 @@ type DDSketchQuantile struct {
 	*ddsketch.DDSketch
 }
 
+const relativeAccuracy = 0.01
+
+var ddsketchPool = sync.Pool{
```

Comment:
Fun fact: the original range vector aggregation is using a pool for series as well.